lib/api/service/authentication_service.dart

```diff
@@ -16,5 +18,9 @@ abstract class AuthenticationServiceImpl extends AuthenticationService {
 
   @override
   @POST('/oauth/token')
-  Future<LoginResponse> logIn(@Body() LoginRequest request);
+  Future<AuthResponse> logIn(@Body() LoginRequest request);
+
+  @override
+  @POST('/oauth/token')
+  Future<AuthResponse> getAuthToken(@Body() RefreshTokenRequest request);
```

Comment:
Should we name it `refreshToken`, which means refresh the access & refresh tokens? `getAuthToken` also fits the log-in case, so I don't think it's a good specific one for this API only.